### PR TITLE
[DLP] Readded inspect with stored infotype and mocked test cases to resolve PR 10556

### PR DESCRIPTION
**Pull request by Avani-Thakker-Crest**
## Description
Mocked stored Info type related test cases solving the revert request: https://github.com/GoogleCloudPlatform/python-docs-samples/pull/10556

This also includes sample for inspect stored infotype since revert request is made. Initial merged PR: https://github.com/GoogleCloudPlatform/python-docs-samples/pull/10456

Fixes #<ISSUE-NUMBER>

Note: Before submitting a pull request, please open an issue for discussion if you are not associated with Google.

## Checklist
- [x] I have followed [Sample Guidelines from AUTHORING_GUIDE.MD](https://github.com/GoogleCloudPlatform/python-docs-samples/blob/main/AUTHORING_GUIDE.md)
- [ ] README is updated to include [all relevant information](https://github.com/GoogleCloudPlatform/python-docs-samples/blob/main/AUTHORING_GUIDE.md#readme-file)
- [x] **Tests** pass:   `nox -s py-3.9` (see [Test Environment Setup](https://github.com/GoogleCloudPlatform/python-docs-samples/blob/main/AUTHORING_GUIDE.md#test-environment-setup))
- [x] **Lint** pass:   `nox -s lint` (see [Test Environment Setup](https://github.com/GoogleCloudPlatform/python-docs-samples/blob/main/AUTHORING_GUIDE.md#test-environment-setup))
- [ ] These samples need a new **API enabled** in testing projects to pass (let us know which ones)
- [ ] These samples need a new/updated **env vars** in testing projects set to pass (let us know which ones)
- [ ] This sample adds a new sample directory, and I updated the [CODEOWNERS file](https://github.com/GoogleCloudPlatform/python-docs-samples/blob/main/.github/CODEOWNERS) with the codeowners for this sample
- [ ] This sample adds a new **Product API**, and I updated the [Blunderbuss issue/PR auto-assigner](https://github.com/GoogleCloudPlatform/python-docs-samples/blob/main/.github/blunderbuss.yml) with the codeowners for this sample
- [x] Please **merge** this PR for me once it is approved

### CODE REVIEW

**Review by leahecole — CHANGES_REQUESTED** [review]
please follow the authoring guide and use pytest fixtures to set up and tear down resources

File: dlp/snippets/stored_infotype_test.py [hunk 1]
```diff
@@ -32,82 +28,85 @@
...
+        GCLOUD_PROJECT,
+        STORED_INFO_TYPE_ID,
+        "The commit was made by gary1998",
+    )
+
+    out, _ = capsys.readouterr()
+    assert "STORED_TYPE" in out
+    assert "Quote: gary1998" in out
+
+    mock_dlp_instance.inspect_content.assert_called_once()
```
Comment (leahecole) [inline dlp/snippets/stored_infotype_test.py:112]: From the authoring guide "Use pytest-style tests and plain asserts. Don't use unittest-style tests or assertX methods"

Comment (Avani-Thakker) [reply dlp/snippets/stored_infotype_test.py:114]: I have updated accordingly. Thank you!

**Comment by Avani-Thakker-Crest** [comment]
> Instead of mocking the call, why not instead create a fixture to delete the resource in DLP?
> 
> Edit: Also adding language from [the authoring guide](https://github.com/GoogleCloudPlatform/python-docs-samples/blob/main/AUTHORING_GUIDE.md) - "Creating mocks for external resources is strongly discouraged. Tests should verify the validity of the sample against the APIs, and not against a mock that embodies assumptions about the behavior of the APIs."

Hi @leahecole Thank you for your comment. An issue was raised and the code work was reverted, as solution to that mocking was required. Issue PR for reference: #10556 . Also, it was decided to mock these tests since only 30 stored infotypes are allowed at a time. We were tearing down the resources, but yet may be due to parallel processing it was crossing the 30 limit. @annadata could help share the context. Kindly let me know how to proceed. Thank you.

**Review by realjordanna — APPROVED** [review]

**Comment by Avani-Thakker-Crest** [comment]
Closing this Pull request as CLA is failing.

**Comment by Avani-Thakker-Crest** [comment]
Closing this Pull request as CLA is failing.

**Pull request closed without merging**